packages/docusaurus-plugin-content-blog/src/plugin-content-blog.d.ts

```diff
@@ -143,6 +143,11 @@ declare module '@docusaurus/plugin-content-blog' {
      * @see {@link BlogPostMetadata.title}
      */
     title?: string;
+    /**
+     * Will override the title collected from the frontMatter.
+     * @see {@link BlogPostMetadata.title_meta}
```

Comment:
comment is wrong, 

this will be used for SEO page metadata and override `BlogPostMetadata.title`